api/apps/api/src/modules/clone/import/domain/import/import.snapshot.ts

```diff
@@ -0,0 +1,14 @@
+import { ArchiveLocation, ResourceKind } from '@marxan/cloning/domain';
+import { ImportComponentSnapshot } from './import.component.snapshot';
+
+export type ImportComponents = Array<
+  ImportComponentSnapshot & { order: number }
+>;
+
+export interface ImportSnapshot {
+  id: string;
+  resourceId: string;
```

Comment:
Should we use ResourceId class?

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/670/commits/de4030574ab0e538d72f8755b472f4b9dd08dc33 thanks for spotting!